src/components/closet/ClosetItemList.tsx

```diff
@@ -72,23 +167,29 @@ export const ClosetItemList = () => {
             </div>
           </div>
         )}
-        {Object.entries(piecesByType).map(([pieceType, pieces]) => (
-          <div key={pieceType}>
-            <h3 className='mt-4 mb-1 text-white bold capitalize text-xl'>
-              {pieceType}
-            </h3>
+        {Object.entries(piecesByType).map(
+          ([pieceType, pieces]) =>
+            filterType !== 'all' &&
```

Comment:
This is causing issues where it will only show the sponsored content, making it never finish loading. Replace `filterType !== 'all' && filterType === pieceType` with `(filterType === 'all' || filterType === pieceType)`

Reply:
g2g